zammad/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 13.0.11
+version: 13.1.0
```

Comment:
Normally changing the default value of a chart is a breaking change.
If an updating user wants to keep the current behaviour he would not know of the change.
Therefore a new major version should be created & upgrade instructions should be added.

Reply:
Changed to 14.0.0, can you check again?